flowmachine/flowmachine/core/server/query_schemas/visited_most_days.py

```diff
@@ -0,0 +1,49 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, you can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marshmallow import fields
+from marshmallow.validate import OneOf
+
+from .base_query_with_sampling import (
+    BaseQueryWithSamplingSchema,
+    BaseExposedQueryWithSampling,
+)
+from flowmachine.features.subscriber.visited_most_days import VisitedMostDays
+
+
+__all__ = [
+    "VisitedMostDaysSchema",
+    "VisitedMostDaysExposed",
+]
+
+
+class VisitedMostDaysExposed(BaseExposedQueryWithSampling):
+    def __init__(self, *, subscriber_locations=None):
+        self.subscriber_locations = subscriber_locations
+
+    @property
+    def _unsampled_query_obj(self):
+        """
+        Return the underlying flowmachine daily_location object.
+
+        Returns
+        -------
+        Query
+        """
+        return VisitedMostDays(
+            subscriber_locations=self.subscriber_locations,
+        )
+
+
+class VisitedMostDaysSchema(
+    StartAndEndField,
```

Comment:
Missing the imports for these

Reply:
My bad, should've removed them. They're not needed since they're already contained in SubscriberLocations.

Reply:
Updated now following slack conversation.